frontend/pages/admin/ManageUsersPage/components/UsersTable/UsersTableConfig.tests.tsx

```
@@ -1,11 +1,36 @@
+  }));
+
+const renderPermissionsCell = (row: IUserTableData) => {
+  const column = generateTableHeaders(jest.fn(), true).find(
+    (c) => c.id === "permissions"
+  );
+  const Cell = column?.Cell as (props: {
+    cell: { value: string };
+    row: { original: IUserTableData };
+  }) => JSX.Element;
+
+  render(<Cell cell={{ value: row.role }} row={{ original: row }} />);
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)